Components/Button/button.css

```diff
@@ -0,0 +1,85 @@
+.btn{
+    border: none;
+    border-radius:5px;    
+    background-color: none;
+    cursor: pointer;
+    font-weight: 400;
+    padding: 1rem 2rem;
+    text-align: center;
+    text-decoration: none;
+}
+
+.btn-primary{
+    background-color: var(--PRIMARY-COLOR);
+    color: var(--TEXT-COLOR);
+}
+
+.btn-secondary{
+    background-color: var(--SECONDARY-COLOR);
+    color: var(--TEXT-COLOR);
+}
+
+
+/* Link */
+.btn-link{
+    text-decoration: none;
+    color: var(--TEXT-COLOR);
+}
+
+
+/* Outline */
+.btn-primary-outline{
+    border: solid var(--PRIMARY-COLOR) 1px;
+    color: var(--PRIMARY-COLOR);
+}
+
+.btn-secondary-outline{
+    border: solid var(--SECONDARY-COLOR) 1px;
```

Comment:
The `border` is the shorthand property of `border-width`, `border-style`, `border-color` in respective order. To maintain consistency in all your CSS try to use the exact way for better readability. Here, it must be `border: 1px solid var(--PRIMARY-COLOR)`.

Reply:
Thankyou for reviewing the code. The changes you mentioned will help me to write consistent code and readable code.